.github/workflows/pull_request.yml

```
@@ -1,28 +1,18 @@
-name: Pull Request
-
+name: "PR Title Checker"
 on:
-  pull_request:
-    branches:
-      - main
+  pull_request_target:
     types:
       - opened
       - edited
       - synchronize
-
-concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: true
+      - labeled
+      - unlabeled
 
 jobs:
-  lint-pr:
-    name: Formatting
+  check:
```

Explicit job name better IMO
```suggestion
  check-pr-title:
```